src/api/app/policies/bs_request_policy.rb

```diff
@@ -7,4 +7,10 @@ def create?
 
     false
   end
+
+  def handle_request?
+    is_target_maintainer = record.is_target_maintainer?(User.session)
```

Comment:
In the context of this policy, the user to be authorized is available through `user`. So with this in mind, this line becomes:

```suggestion
    is_target_maintainer = record.is_target_maintainer?(user)
```

Same for the next line.